@here/harp-lrucache/lib/LRUCache.ts

```diff
@@ -159,6 +153,26 @@ export class LRUCache<Key, Value> {
         this.evict();
     }
 
+    /**
+     * Updates the size of all elements in this cache. If their aggregated size is larger than the
+     * capacity, items will be evicted until the cache shrinks to fit the capacity.
+     */
+    checkCapacity(): void {
```

Comment:
This needs a better name as it actually mutates the state. `shrinkToSize` or something?

Reply:
can do

Reply:
implemented as requested